src/edu/washington/escience/myria/accessmethod/AccessMethod.java

```diff
@@ -154,6 +154,12 @@ public void init() throws DbException {
   public abstract void dropTableIfExists(RelationKey relationKey) throws DbException;
 
   /**
+   * @param relationDelete the relation to drop
+   * @throws DbException if there is an error dropping the table.
+   */
+  public abstract void dropTableIfExistsCascade(RelationKey relationDelete) throws DbException;
```

Comment:
Please rename the `relationDelete` param to `relationKey` for consistency with the other methods and the `JdbcAccessMethod` implementation.